pl_bolts/models/vision/unet.py

```diff
@@ -67,38 +67,35 @@ def forward(self, x):
         return self.layers[-1](xi[-1])
 
 
-@under_review()
 class DoubleConv(nn.Module):
-    """[ Conv2d => BatchNorm (optional) => ReLU ] x 2."""
+    """[ Conv2d => BatchNorm => ReLU ] x 2."""
 
     def __init__(self, in_ch: int, out_ch: int):
         super().__init__()
         self.net = nn.Sequential(
-            nn.Conv2d(in_ch, out_ch, kernel_size=3, padding=1),
+            nn.Conv2d(in_ch, out_ch, kernel_size=3, padding=1, bias=False),
```

Comment:
Why are you explicitly removing bias?

Reply:
Bias of convolution before batch normalization can be removed because the beta term in batch normalization can replace the role of bias(https://discuss.pytorch.org/t/any-purpose-to-set-bias-false-in-densenet-torchvision/22067), and `nn.Conv2d` has default value of `bias=True`.

Reply:
Thank you for the pointer! It's a piece of information I somehow had stored very far in my head, but just didn't connect it with the proper reason why is it done here :) You're absolutely right :zap: